--- src/base/linuxthreads.cc	2012-07-23 02:21:30.946086243 +0800
+++ src/base/linuxthreads.cc	2012-02-03 05:36:23.000000000 +0800
@@ -193,7 +193,7 @@
 /* Signal handler to help us recover from dying while we are attached to
  * other threads.
  */
-static void SignalHandler(int signum, siginfo_t *si, void *data) {
+static void SignalHandler(int signum, siginfo *si, void *data) {
   if (sig_pids != NULL) {
     if (signum == SIGABRT) {
       while (sig_num_threads-- > 0) {
